docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -6269,7 +6269,7 @@ end
 |===
 | Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
 
-| Enabled
+| Disabled
```

Comment:
The documentation is automatically generated at release, so no updates are needed.

Reply:
Thanks. I've reverted this change https://github.com/rubocop/rubocop/pull/13788/commits/edbbb876beebd897c0a2443728215d4faa15b5a6